InterviewView.swift

```diff
@@ -0,0 +1,39 @@
+
+import UIKit
+
+class ViewController: UIViewController {
+    var output: ProfileViewOutput?
+    let resultButton = UILabel()
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        resultButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        resultButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        resultButton.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        resultButton.widthAnchor.constraint(equalToConstant: 100).isActive = true
+        resultButton.heightAnchor.constraint(equalToConstant: 60).isActive = true
+        view.addSubview(resultButton)
```

Comment:
Adding button as a subview at each view appearing, fix it and add only after view is loaded